source/install/config-proxy-nginx.rst

```diff
@@ -0,0 +1,88 @@
+.. _config-proxy-nginx:
+
+Configuring NGINX as a proxy for Mattermost Server
+==================================================
+
+**To configure NGINX as a proxy**
```

Comment:
I added a short description so that the line makes more sense. See commit https://github.com/JeffSchering/docs/commit/3b73c7bd9dffe1adbd82f29cba48710ffad64915